interface/modules/zend_modules/public/index.php

```diff
@@ -35,6 +35,11 @@
     }
 }
 
+if (php_sapi_name() === 'cli') {
+    $ignoreAuth = true;
+    $_GET['site'] = 'default';
```

Comment:
Would be much better to not hard-code the site variable. Recommend allowing to set this when call via command line (if it is not set at command line, then would set it to 'default'. This then means that folks that are using multisite can use it.

Reply:
Hi @bradymiller. Hard-code has been removed.